src/components/Item/NotSupportedItem/Item.js

```diff
@@ -1,11 +1,17 @@
 import React, { Fragment } from 'react';
 import PropTypes from 'prop-types';
+import i18n from '@dhis2/d2-i18n';
 import ItemHeader from '../ItemHeader';
 import NotInterestedIcon from '@material-ui/icons/NotInterested';
 
 const NotSupportedItem = props => (
     <Fragment>
-        <ItemHeader title={`Item type not supported: ${props.item.type}`} />
+        <ItemHeader
+            title={i18n.t('Item type "{{type}}" not supported', {
```

Comment:
Are the double-quotes around {{type}} intentional? They will show up in the result, as: Item type "SomeType" not supported

Reply:
That was intentional, yes.